easybuild/framework/easyconfig/templates.py

```diff
@@ -201,6 +203,13 @@ def template_constant_dict(config, ignore=None, skip_lower=None, toolchain=None)
     # set 'arch' for system architecture based on 'machine' (4th) element of platform.uname() return value
     template_values['arch'] = platform.uname()[4]
 
+    # set 'sysroot' based on sysroot build option
+    sysroot = build_option('sysroot')
+    if sysroot is None:
+        template_values['sysroot'] = ""
+    else:
+        template_values['sysroot'] = sysroot
```

Comment:
```suggestion
    # set 'sysroot' template based on 'sysroot' configuration option, using empty string as fallback
 value
    template_values['sysroot'] = build_option('sysroot') or ''

Reply:
Elegant and short his one. We will have to rely on None evaluating to False in this context, but I think it's safe enough to assume Python won't change that anytime soon (famous last words :D)